CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -246,4 +246,48 @@ public static function getFinancialAccountForFinancialTypeByRelationship($financ
     return Civi::$statics[__CLASS__]['entity_financial_account'][$financialTypeID][$relationTypeId];
   }
 
+  /**
+   * check if financial type has Deferred Revenue Account is relationship
+   * with Financial Account
+   *
+   * @param $financialTypeId Integer
+   *
+   * @return string
+   *
+   */
+  public static function validateFinancialType($financialTypeId, $entityID = NULL, $entity = NULL) {
+    if (!$financialTypeId) {
```

Comment:
It does, when updating Event, MembershipType etc we might not send financial_type_id in $params via api or BAO function. The function should return false in such situation.